### PR TITLE
nf-yellowdog 1.0.0

### PR DESCRIPTION
**Pull request by rupert-madden-abbott**

### CODE REVIEW

File: plugins.json [hunk 1]
```diff
@@ -2445,5 +2445,17 @@
...
       }
     ]
+  },
+  {
+    "id": "yellowdog",
+    "releases": [
+      {
+        "version": "1.0.0",
+        "url": "https://github.com/yellowdog/nextflow-plugin-public/releases/download/1.0.0/yellowdog-1.0.0.zip",
+        "date": "2024-02-08T?14:36:00+00:00",
```
Comment (bentsherman) [inline plugins.json:2455]:
```suggestion
        "date": "2024-02-08T14:36:00+00:00",
```

Comment (bentsherman) [reply plugins.json:2455]: Not sure how the question mark got here but I think it is breaking the validation rule

**Comment by bentsherman** [comment]
Thanks for publishing your plugin! If you can fix the validation error then I'll merge it.

Also, it would be great if you could add some documentation to the README of your plugin repo (or link to docs if you have them elsewhere). The plugin repo is a common entrypoint for interested users.

**Comment by rupert-madden-abbott** [comment]
@bentsherman Thanks very much for spotting the date error. That has now been fixed.

Yes we can absolutely link to our documentation from the README in our repo. We will get that updated shortly.

**Pull request merged**